plugins/SLURM-Integration/ufm_slurm_utils.py

```diff
@@ -106,9 +107,11 @@ def get_conf_parameter_value(self, conf_param_name):
         :return None: in case the parameter name was not found in ufm_slurm.conf file.
         """
         conf_file_path = self.getSlurmConfFile()
-        conf_mode = os.stat(conf_file_path).st_mode
-        if conf_mode & 0o77 != 0:
-            logging.error(f"Permissions for configuration file {conf_file_path} are too open: {oct(conf_mode & 0o777)}")
+        conf_mode = stat.S_IMODE(os.lstat(conf_file_path).st_mode)
```

Comment:
please remove this code / section - we don't need it as we set the right file permission upon the installation of the slurm integration.